java-generator/it/src/it/plugin/gradle/simple/expected/ActiveMQArtemisSpec.expected

```diff
@@ -0,0 +1,169 @@
+package io.amq.broker.v1beta1;
```

Comment:
instead of checking the content of the generated files would it be possible to verify in a test that the generated sources have been compiled and are available on the classpath?

Reply:
This should be fixed now.